lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -55,5 +79,80 @@ export default TestModule.extend({
 
       return subject.$.apply(subject, arguments);
     };
+  },
+
+  setupComponentIntegrationTest: function() {
+    var self = this;
+    var context = this.context;
+    context.dispatcher = Ember.EventDispatcher.create();
+    context.dispatcher.setup({}, '#ember-testing');
+    this.actionHooks = {};
+
+    context.render = function(template) {
+      if (!template) {
+        throw new Error("in a component integration test you must pass a template to `render()`");
+      }
+      if (Ember.isArray(template)) {
+        template = template.join('');
+      }
+      if (typeof template === 'string') {
+        template = Ember.Handlebars.compile(template);
+      }
+      self.view = Ember.View.create({
```

Comment:
Can we call this `component` (I dream of an all component world, and don't want to be stuck using the term `view` when they are essentially a private internal concept in Ember 2.0).